runtime/util_core/j9argscan.c

```diff
@@ -385,6 +385,97 @@ scan_hex_caseflag_u64(char **scan_start, BOOLEAN uppercaseAllowed, uint64_t* res
 	return bits;
 }
 
+/**
+ * Scan the next unsigned number off of the argument string, and parses for memory sizes.
+ * Specify the size in GiBs, MiBs, or KiBs (with G,g,M,m,K,k suffixes) or in bytes (no suffix).
+ * @param[in] scan_start The string to be scanned
+ * @param[out] result The result
+ * @return returns 0 on success, 1 if the argument string is not a number, or 2 if overflow occurs.
+ */
+uintptr_t
+scan_u64_memory_size(char **scan_start, uint64_t* result)
+{
+	uintptr_t rc = scan_u64(scan_start, result);
+
+	/* Report any errors from scan */
+	if(0 != rc) {
+		return rc;
+	}
+
```

Comment:
this helper should be updated to handle `T` and `t` as well

Reply:
Right U64 should be able to support T/t as well. 
See: https://github.com/AlenBadel/openj9/commit/0de782248cdeb7466aa17e33809f85ff7c6b7f3a